### PR TITLE
[12.0] [ADD] project_category: Added some fields in the Project type model.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**
**Project Type Model Enhancement**

### CODE REVIEW

**Comment by Chandresh-SerpentCS** [comment]
@dreispt Request for review the PR.

**Review by HaraldPanten — APPROVED** [review]

**Review by alexey-pelykh — COMMENTED** [review]
I don’t really understand value of the module except for adding checkboxes

**Comment by alexey-pelykh** [comment]
Or how name of the module corresponds to content

File: project_category/models/project_type.py [hunk 1]
```diff
@@ -38,6 +38,18 @@ class ProjectType(models.Model):
     task_ok = fields.Boolean(
         string='Can be applied for tasks'
     )
+    is_billable = fields.Boolean(
```
Comment (alexey-pelykh) [inline project_category/models/project_type.py:41]: Can’t be this deduced from sale order link?

**Review by alexey-pelykh — COMMENTED** [review]
Cc @pedrobaeza calling fo extra opinion

**Comment by dreispt** [comment]
@alexey-pelykh 
I prepared these requirements for Chandresh. I realize now I should have created an RFQ Issue to clarify the goals.
The requirement behind this is to enable Project time analysis.

- Billable projects are "productive time", they mean income, and the other mean expenses.
- Budgeted projects are internal projects that have a budget assigned to. So they are also "productive time", but they do not generated income.

Maybe the "Is internal" flag is redundant.

Thoughts?

**Comment by alexey-pelykh** [comment]
Hm, thanks for the information. I got the idea, yet I don't understand how this implementation would work (on it's own) with other OCA modules. It feels like a valid business case, yet unlikely applicable (in current implementation) as-is to other deployments. Billable projects (well, 100% expenses projects or with some profit) is doable via analytic accounts with ease, for much stronger analysis capabilities. Budgeted projects are investment projects, in other words. Likewise, analytic accounts [with parenting] could be a way of differentiating cost centers. In current implementation I'm leaning towards not supporting this module: too broad name, too detached implementation, low value (IMHO).

**Comment by dreispt** [comment]
Adding these field would lay the ground for this to be used in other places. Alone adds low value, but is as enabler for features adding actual values, such as dashboards or List view filters/groups.

IIUC your suggestion is to use Parent Analytic Accounts? This greatly limits your flexibility to freely setup your Analytic tree. Also, I'm having trouble finding this field on the default UI.

Now, I noticed we have a Analytic Account "Group" field, that could be used for this:
image

**Comment by alexey-pelykh** [comment]
> Adding these field would lay the ground for this to be used in other places. Alone adds low value, but is as enabler for features adding actual values, such as dashboards or List view filters/groups.

Absolutely, if this module would've added these dashboards etc. - then it would've made sense.

Does Analytic Tags/Groups solve your case?

**Comment by dreispt** [comment]
We're solving this is a different way - just using the Project Types, thanks.

**Pull request closed without merging**